src/main/kotlin/com/odoo/odools/OdooLSPathConfigurable.kt

```diff
@@ -44,13 +46,21 @@ class OdooLSPathConfigurable : Configurable {
             <html>
                 <body style='width: 500px; color: gray;'>
                     The plugin will use this path to install needed resources and logs.<br>
+                    To configure the settings in <code>odools.toml</code>, please consult the wiki linked below:
```

Comment:
OdooLS is configures through configuration files. To create one, please refer to our wiki: